packages/shared/constants/error.js

```diff
@@ -0,0 +1,2 @@
+export const CANNOT_COMPLETE_ABANDONED_PLAN = "Cannot complete management plan with an abandon date"
```

Comment:
I created this file for error strings returned by the API that need to be validated by the UI to display an error message. This way if on the BE side we decide to change the contents of the string for any reason, the UI will get the updated string to compare to.